nmt.go

```diff
@@ -421,7 +421,7 @@ func (n *NamespacedMerkleTree) foundInRange(nID namespace.ID) (found bool, start
 	foundRng, found := n.namespaceRanges[string(nID)]
 	// XXX casting from uint64 to int is kinda crappy but nebolousLabs' range
 	// proof api requires int params only to convert them to uint64 ...
```

Comment:
[question] is this comment residual now b/c there isn't an explicit type cast on line 424?

```suggestion
```